src/ws/cockpitsshtransport.h

```diff
@@ -33,15 +33,17 @@ G_BEGIN_DECLS
 typedef struct _CockpitSshTransport        CockpitSshTransport;
 typedef struct _CockpitSshTransportClass   CockpitSshTransportClass;
 
-GType               cockpit_ssh_transport_get_type              (void) G_GNUC_CONST;
+GType               cockpit_ssh_transport_get_type                (void) G_GNUC_CONST;
```

Comment:
Typically we don't reindent the headers, just add another section down below that is further indented.